src/js/apps/patients/worklist/worklist_state.js

```diff
@@ -110,6 +110,10 @@ export default Backbone.Model.extend({
       };
     }
 
+    if (relativeDate === 'alltime') {
+      return null;
```

Comment:
Just return `{}` here then the `|| {}` aren't necessary.

Reply:
Yup that's a better way to do it. Just pushed that update.